src/thunderbots/software/test/ai/game_state_play_selection.cpp

```diff
@@ -0,0 +1,52 @@
+#include <gtest/gtest.h>
+#include "ai/ai.h"
+#include "ai/world/world.h"
+#include "test/test_util/test_util.h"
+
+class GameStatePlaySelectionTest : public ::testing::Test, public ::testing::WithParamInterface<RefboxGameState> {
+public:
+protected:
+    void SetUp() override {
+        world.mutableField() = ::Test::TestUtil::createSSLDivBField();
+
+        Robot robot_0(0, Point(-1.1, 1), Point(), Angle::zero(), AngularVelocity::zero(),
+                      Timestamp::fromSeconds(0));
+        Robot robot_1(1, Point(2, 0.81), Point(), Angle::zero(), AngularVelocity::zero(),
+                      Timestamp::fromSeconds(0));
+        Robot robot_2(2, Point(0, 5.0), Point(), Angle::zero(), AngularVelocity::zero(),
+                      Timestamp::fromSeconds(0));
+        world.mutableFriendlyTeam().updateRobots({robot_0, robot_1, robot_2});
+
+        Robot enemy_robot_0(0, Point(-1.1, 1), Point(), Angle::zero(), AngularVelocity::zero(),
+                      Timestamp::fromSeconds(0));
+        Robot enemy_robot_1(1, Point(2, 0.81), Point(), Angle::zero(), AngularVelocity::zero(),
+                      Timestamp::fromSeconds(0));
+        Robot enemy_robot_2(2, Point(0, 5.0), Point(), Angle::zero(), AngularVelocity::zero(),
+                      Timestamp::fromSeconds(0));
```

Comment:
Maybe we should invert the `x` values for friendlies so the robots aren't on top of each other? :laughing:

Reply:
fixed